### PR TITLE
Pioneer 15b gliders

### PR DESCRIPTION
**Pull request by cdobs**
Recovered:
CP05MOAS-GL380 R00010
CP05MOAS-GL387 R00008

Telemetered:
CP05MOAS-GL339 D00012
CP05MOAS-GL376 D00013
CP05MOAS-GL379 D00011
CP05MOAS-GL559 D00001

### CODE REVIEW

File: CP05MOAS-GL339/CP05MOAS-GL339_D00012_ingest.csv [hunk 1]
```diff
@@ -0,0 +1,6 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL339/D00012/merged-from-glider/cp_*.mrg,CP05MOAS-GL339-00-ENG000000,telemetered,Available,
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL339/D00012/merged-from-glider/cp_*.mrg,CP05MOAS-GL339-02-FLORTM000,telemetered,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL339/D00012/merged-from-glider/cp_*.mrg,CP05MOAS-GL339-03-CTDGVM000,telemetered,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL339/D00012/merged-from-glider/cp_*.mrg,CP05MOAS-GL339-04-DOSTAM000,telemetered,Available,
+mi.dataset.driver.moas.gl.parad.parad_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL339/D00012/merged-from-glider/cp_*.mrg,CP05MOAS-GL339-05-PARADM000,telemetered,Available,
```
Comment (snwhite) [inline CP05MOAS-GL339/CP05MOAS-GL339_D00012_ingest.csv:6]: Looks good.

File: CP05MOAS-GL376/CP05MOAS-GL376_D00013_ingest.csv [hunk 2]
```diff
@@ -0,0 +1,6 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL376/D00013/merged-from-glider/cp_*.mrg,CP05MOAS-GL376-00-ENG000000,telemetered,Available,
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL376/D00013/merged-from-glider/cp_*.mrg,CP05MOAS-GL376-02-FLORTM000,telemetered,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL376/D00013/merged-from-glider/cp_*.mrg,CP05MOAS-GL376-03-CTDGVM000,telemetered,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL376/D00013/merged-from-glider/cp_*.mrg,CP05MOAS-GL376-04-DOSTAM000,telemetered,Available,
+mi.dataset.driver.moas.gl.parad.parad_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL376/D00013/merged-from-glider/cp_*.mrg,CP05MOAS-GL376-05-PARADM000,telemetered,Available,
```
Comment (snwhite) [inline CP05MOAS-GL376/CP05MOAS-GL376_D00013_ingest.csv:6]: Looks good.

File: CP05MOAS-GL379/CP05MOAS-GL379_D00011_ingest.csv [hunk 3]
```diff
@@ -0,0 +1,6 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL379/D00011/merged-from-glider/cp_*.mrg,CP05MOAS-GL379-00-ENG000000,telemetered,Available,
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL379/D00011/merged-from-glider/cp_*.mrg,CP05MOAS-GL379-02-FLORTM000,telemetered,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL379/D00011/merged-from-glider/cp_*.mrg,CP05MOAS-GL379-03-CTDGVM000,telemetered,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL379/D00011/merged-from-glider/cp_*.mrg,CP05MOAS-GL379-04-DOSTAM000,telemetered,Available,
+mi.dataset.driver.moas.gl.parad.parad_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL379/D00011/merged-from-glider/cp_*.mrg,CP05MOAS-GL379-05-PARADM000,telemetered,Available,
```
Comment (snwhite) [inline CP05MOAS-GL379/CP05MOAS-GL379_D00011_ingest.csv:6]: Looks good.

File: CP05MOAS-GL559/CP05MOAS-GL559_D00001_ingest.csv [hunk 4]
```diff
@@ -0,0 +1,6 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL559/D00001/merged-from-glider/cp_*.mrg,CP05MOAS-GL559-00-ENG000000,telemetered,Available,
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL559/D00001/merged-from-glider/cp_*.mrg,CP05MOAS-GL559-02-FLORTM000,telemetered,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL559/D00001/merged-from-glider/cp_*.mrg,CP05MOAS-GL559-03-CTDGVM000,telemetered,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL559/D00001/merged-from-glider/cp_*.mrg,CP05MOAS-GL559-04-DOSTAM000,telemetered,Available,
+mi.dataset.driver.moas.gl.parad.parad_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL559/D00001/merged-from-glider/cp_*.mrg,CP05MOAS-GL559-05-PARADM000,telemetered,Available,
```
Comment (snwhite) [inline CP05MOAS-GL559/CP05MOAS-GL559_D00001_ingest.csv:6]: Looks good.

File: CP05MOAS-GL380/CP05MOAS-GL380_R00010_ingest.csv [hunk 5]
```diff
@@ -0,0 +1,7 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL380/R00010/merged/cp_*.mrg,CP05MOAS-GL380-00-ENG000000,recovered_host,Available,
+mi.dataset.driver.moas.gl.adcpa.adcpa_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL380/R00010/dvl/*.PD0,CP05MOAS-GL380-01-ADCPAM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL380/R00010/merged/cp_*.mrg,CP05MOAS-GL380-02-FLORTM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL380/R00010/merged/cp_*.mrg,CP05MOAS-GL380-03-CTDGVM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL380/R00010/merged/cp_*.mrg,CP05MOAS-GL380-04-DOSTAM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.parad.parad_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL380/R00010/merged/cp_*.mrg,CP05MOAS-GL380-05-PARADM000,recovered_host,Available,
```
Comment (snwhite) [inline CP05MOAS-GL380/CP05MOAS-GL380_R00010_ingest.csv:7]: Looks good.

File: CP05MOAS-GL387/CP05MOAS-GL387_R00008_ingest.csv [hunk 6]
```diff
@@ -0,0 +1,7 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL387/R00008/merged/cp_*.mrg,CP05MOAS-GL387-00-ENG000000,recovered_host,Available,
+mi.dataset.driver.moas.gl.adcpa.adcpa_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL387/R00008/dvl/*.PD0,CP05MOAS-GL387-01-ADCPAM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL387/R00008/merged/cp_*.mrg,CP05MOAS-GL387-02-FLORTM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL387/R00008/merged/cp_*.mrg,CP05MOAS-GL387-03-CTDGVM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL387/R00008/merged/cp_*.mrg,CP05MOAS-GL387-04-DOSTAM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.parad.parad_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL387/R00008/merged/cp_*.mrg,CP05MOAS-GL387-05-PARADM000,recovered_host,Available,
```
Comment (snwhite) [inline CP05MOAS-GL387/CP05MOAS-GL387_R00008_ingest.csv:7]: Looks good.

**Review by snwhite — APPROVED** [review]

File: CP05MOAS-GL339/CP05MOAS-GL339_D00012_ingest.csv (hunk 1 above)
Comment (pbrickley) [inline CP05MOAS-GL339/CP05MOAS-GL339_D00012_ingest.csv:6]: OK

File: CP05MOAS-GL376/CP05MOAS-GL376_D00013_ingest.csv (hunk 2 above)
Comment (pbrickley) [inline CP05MOAS-GL376/CP05MOAS-GL376_D00013_ingest.csv:6]: OK

File: CP05MOAS-GL379/CP05MOAS-GL379_D00011_ingest.csv (hunk 3 above)
Comment (pbrickley) [inline CP05MOAS-GL379/CP05MOAS-GL379_D00011_ingest.csv:6]: OK

File: CP05MOAS-GL380/CP05MOAS-GL380_R00010_ingest.csv (hunk 5 above)
Comment (pbrickley) [inline CP05MOAS-GL380/CP05MOAS-GL380_R00010_ingest.csv:7]: OK

File: CP05MOAS-GL387/CP05MOAS-GL387_R00008_ingest.csv (hunk 6 above)
Comment (pbrickley) [inline CP05MOAS-GL387/CP05MOAS-GL387_R00008_ingest.csv:7]: OK

File: CP05MOAS-GL559/CP05MOAS-GL559_D00001_ingest.csv (hunk 4 above)
Comment (pbrickley) [inline CP05MOAS-GL559/CP05MOAS-GL559_D00001_ingest.csv:6]: OK (noting this was a conversion from global. Previously gi_559.)

**Review by pbrickley — APPROVED** [review]

**Pull request merged**